### DISCUSSION TITLE
I think it's time to support AudioBookShelf

### DISCUSSION THREAD

**Discussion by sectary-want-holly** [PaulWoitaschek/Voice#2666] [Ideas]
As per $subject. I see earlier discussion [here](https://github.com/PaulWoitaschek/Voice/discussions/2385) in April, but now is the time to re-request ABS support.  I see Jellyfind support was implemented, so we are trending in the right direction. 

The reality is that there is now no better hosting platform for audiobooks than ABS currently. It's superb, yet suffers from a first party app that's not widely available on iOS and third party ones which are 'OK'. ABP could gain considerably more adoption if they become compatible with ABS.

**Comment by MikotoMisaka10032** [comment]
I would also love to see Audiobookshelf support in Voice. \
It would be a fantastic addition.
